### DISCUSSION TITLE
🆕  Improved Issue creation experience for GitHub Mobile 📱

### DISCUSSION THREAD

**Discussion by Akash1134** [community/community#60854] [Mobile]
![Screenshot 2023-07-14 at 6 26 55 PM](https://github.com/community/community/assets/95834130/2ac254ec-dee2-4c4c-949b-cb244d7961f4)

## 🎉 Introducing a new way of adding metadata while creating issues on GitHub Mobile 📱 

We are excited to enhance the Issue creation experience for GitHub Mobile by[ introducing a comprehensive Property Bar](https://github.blog/changelog/2023-07-13-improved-issue-creation-experience-for-github-mobile/). This feature allows you to conveniently add assignees, labels, milestones, and projects while creating Issues on GitHub Mobile. ⚡ 

This powerful ✊ addition ensures that you have the necessary tools at your fingertips to create your issues with all relevant metadata even quicker than before. ✅ 

[Read more about GitHub Mobile](https://github.com/mobile) and share your feedback about this feature in comments to help us improve. 🤝

**Comment by Akash1134** [comment]
🚀

**Comment by adiroiban** [comment]
This is very helpful.
Looking forward to have the same features even editing an existing issue.

It's strange to have this feature available only at issue creation time.

Thanks

**Reply by ManishParashar0** [reply]
let me check

**Reply by sinshade** [reply]
T

**Reply by sinshade** [reply]
Creation

**Comment by ManishParashar0** [comment]
i cant uderstand
